src/controllers/diff.controller.ts

```diff
@@ -0,0 +1,41 @@
+
+import { NextFunction, Request, Response, Router } from 'express';
+import Ajv from 'ajv';
+
+import { diff } from "@asyncapi/diff";
+
+import { Controller} from '../interfaces';
+
+import { documentValidationMiddleware } from '../middlewares/document-validation.middleware';
+
+
+export class DiffController implements Controller {
+    public basepath = '/diff';
+    private ajv: Ajv;
+
+    private async difference(req: Request, res: Response, next: NextFunction) {
+        try {
+            const output = await diff(req.body.asyncapi, req.body.other);
+            res.status(200).json(output);
+        } catch (err) {
+            return next(err);
```

Comment:
maybe we should make sure we call `next` with an instance of `ProblemException`, wdyt?
You can check out other controllers to see how they are implemented, and do something similar 👍